msm-sitemap.php

```diff
@@ -674,6 +674,9 @@ public static function build_root_sitemap_xml( $year = false ) {
 		// Sometimes duplicate sitemaps exist, lets make sure so they are not output
 		$sitemaps = array_unique( $sitemaps );
 
+		// Filter daily sitemaps from the index by date.
```

Comment:
This should be [properly documented](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#4-hooks-actions-and-filters).

Reply:
@GaryJones This is documented now. What is next on the path to merger?